### PR TITLE
style: add icons in transaction details

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/containers/AccountHistory/Table.tsx [hunk 1]
```diff
@@ -39,47 +50,57 @@ export const ActivityTable = ({ rows = [] }: ActivityTableProps) => {
...
-                    query: { chain: chainPath, tx: row.txHash },
-                  }}
-                >
-                  <Image src={detailsIcon} alt='Open transaction details page' />
-                </Link>
-              </TableCell>
-            </STableRow>
-          ))}
-        </STableBody>
+        {rows.length !== 0 && (
```
Comment (turtlemoji) [inline src/containers/AccountHistory/Table.tsx:53]:
```suggestion
        {!!rows.length && (
```
this works?

File: src/containers/AccountHistory/Table.tsx [hunk 2]
```diff
@@ -39,47 +50,57 @@ export const ActivityTable = ({ rows = [] }: ActivityTableProps) => {
...
+                    <Image src={detailsIcon} alt='Open transaction details page' />
+                  </Link>
+                </TableCell>
+              </STableRow>
+            ))}
+          </STableBody>
+        )}
       </Table>
+
+      {rows.length === 0 && (
```
Comment (turtlemoji) [inline src/containers/AccountHistory/Table.tsx:97]:
```suggestion
      {!rows.length && (
```
and this?

**Review by turtlemoji — APPROVED** [review]
pre-approved left some comments

File: src/containers/AccountHistory/Table.tsx (hunk 1 above)
Comment (0xArdy) [reply src/containers/AccountHistory/Table.tsx:53]: yes, i'll change it in the next pr if u dont mind, so i can merge this pr now

**Pull request merged**